corpus/templates/pages/sig.html

```diff
@@ -2,11 +2,11 @@
 {% load static %}
 
 {% block title %}
-Home
+{{sig.name}}
 {% endblock %}
 
 {% block style %}
-<link rel="stylesheet" href="{% static 'css/index.css' %}" />
+<!-- <link rel="stylesheet" href="{% static 'css/index.css' %}" /> -->
```

Comment:
Remove the style block entirely if you're not using custom styles.

Reply:
Yeah was testing it out that time and had forgotten to remove the block, fixed now